### DISCUSSION TITLE
RELEASE 4.4.0

### DISCUSSION THREAD

**Discussion by Siemienik** [exceljs/exceljs#2351] [General]
Hello everyone 👋,

I'm excited to announce that I've obtained permissions for `exceljs` on npm. This paves the way for us to continue and revitalize the development of this project - Thank you @guyonroche .

I'm planning to roll out version 4.4.0 soon, which will encompass the following changes:
https://github.com/exceljs/exceljs/compare/v4.3.0...master

A big shoutout to every single contributor out there. Your contributions, large or small, have been invaluable, and I genuinely want to express my appreciation. You're all amazing!

Additionally, I'd like to remind everyone that I have been organizing MergeFests from time to time. With the renewed momentum, I'm optimistic about hosting these events more regularly. During these sessions, we'll review and discuss pending Pull Requests. More information may be found on my [Discord](https://discord.gg/siema).

I urge everyone, new contributors and veterans alike, to join hands and collaborate. Together, we can elevate `exceljs` to new heights.

Warm regards,
Pawel Siemienik

---

[![SiemaTeam](https://discordapp.com/api/guilds/976854442009825321/widget.png?style=banner2)](https://discord.gg/siema)

**Comment by skypesky** [comment]
:+1:

**Comment by gltjk** [comment]
Great news! Looking forward to the next release :grinning:

**Comment by L-U-C-K-Y** [comment]
@Siemienik fantastic news, much appreciated, wish you all the best

**Comment by Siemienik** [comment]
Changelog:

* #1815: Thanks, @andreykrupskii!
* #1576: Thanks, @drdmitry!
* #1849: Thanks, @bno1!
* #1670: Grateful for you, @thambley!
* #1850: Great job, @ikzhr!
* #1846: Thank you, @cclauss!
* #1831: Thanks, @bno1!
* #1856: Grateful for you, @Siemienik!
* #1895: Thank you, @ValerioSevilla!
* #1677: Much appreciated, @xjrcode!
* #1700: Great job, @joeldenning!
* #1733: Thanks, @alexbjorlig!
* #1701: Thank you, @skypesky!
* #2242: Grateful for you, @Siemienik!
* #2223: Grateful for you, @hfhchan-plb!
* #2126: Thank you, @tkm-kj!
* #1756: Great job, @pauliusg!
* #2186: Thank you, @wulfsolter!
* #1982: Grateful for you, @DemoJj!
* #1982: Great job, @DemoJj!
* #1934: Great job, @ZyqGitHub1!
* #2211: Thanks, @jarrod-cocoon!
* #2208: Thanks, @HugoP27!
* #2244: Great job, @zurmokeeper!
* #2257: Thanks, @Siemienik!
* #2140: Much appreciated, @babu-ch!
* #2222: Thanks, @cpaiyueyue!
* #2222: Thank you, @cpaiyueyue!
* #2327: Much appreciated, @albeniraouf!
* #2354: Thanks, @Siemienik!
* #2350: Grateful for you, @NiklasPor!
* #2320: Thanks, @hfhchan-plb!
* #2324: Grateful for you, @justintunev7!
* #2309: Grateful for you, @gltjk!
* #2284: Grateful for you, @damingerdai!
* #2281: Great job, @ytjmt!

**Reply by Siemienik** [reply]
PR with README.md update : #2555

**Comment by VivionLin** [comment]
I'm looking forward to the new ignore nodes feature to load my files that have many data validation rules~
 @Siemienik Thanks for your work !!

**Comment by Siemienik** [comment]
Released: https://github.com/exceljs/exceljs/releases/tag/v4.4.0

**Comment by iwangbowen** [comment]
I really appreciate your hard work and hope this project  can achieve greater success.
